Dockerfile.current

```diff
@@ -64,10 +61,11 @@ RUN git clone "https://github.com/dylan-lang/opendylan.git" --recursive \
  && make PATH="/build/release/bin:${PATH}" \
  && make install
 
-# Check out, build and install dylan-tool
-RUN git clone "https://github.com/dylan-lang/dylan-tool.git" --recursive \
- && cd dylan-tool \
- && git checkout ${DYLANTOOL_VERSION} \
+# Check out, build and install 'deft'
+RUN git clone "https://github.com/dylan-lang/deft.git" --recursive \
```

Comment:
`--recursive` is no longer needed (but doesn't hurt)

But yeah, I think OD's `make install` should already be installing Deft.